src/Slider.js

```diff
@@ -24,11 +20,6 @@ class Slider extends React.Component {
             <input
                 className={classes}
                 type="range"
-                min={min}
-                max={max}
-                value={value}
-                tabIndex="0"
```

Comment:
Doesn't tabindex have a default of 0 anyhow? I'd have to check into it, but I'm pretty sure that's the case